wpilibc/src/main/native/include/frc/commands/InstantCommand.h

```diff
@@ -28,17 +31,59 @@ class InstantCommand : public Command {
   explicit InstantCommand(const wpi::Twine& name);
 
   /**
-   * Creates a new InstantCommand with the given name.
+   * Creates a new InstantCommand that depends on the given Subsystem.
+   *
+   * @param requirement The subsystem that the command requires
+   */
+  explicit InstantCommand(Subsystem& requirement);
+
+  /**
+   * Creates a new InstantCommand with the given name and requirement.
    *
    * @param name        The name for this command
    * @param requirement The subsystem that the command requires
    */
   InstantCommand(const wpi::Twine& name, Subsystem& requirement);
 
+  /**
+   * Create a command that calls the given function when run.
+   *
+   * @param func The func to take when the command is run.
```

Comment:
"The func to take" is inaccurate because _initialize() doesn't take the function; it runs the function. I'd say "The function to run when Command::Initialize() is called." for C++ and "The function to run when Command.initialize() is called." for Java. Same applies to the other constructors.